### DISCUSSION TITLE
More than 5 Loras loaded at same time

### DISCUSSION THREAD

**Discussion by enternalsaga** [lllyasviel/Fooocus#3040] [Q&A] [answered]
Just noticed the lora was updated to work right in prompt, however it is only function as long as there are less than 5 loras were chosen in checkboxes. Given the fact that many Loras can add in one prompt in Pony examples, can we add more than 5 Loras in one generation?
Thank you.

**Comment by Rogala** [comment, marked as answer]
Add a line to **config.txt** with the amount you need, watch out for commas at the end of each line, the last line should not have any.
`“default_max_lora_number”: 1...N`

**Reply by enternalsaga** [reply]
great answer, thank you!

**Reply by Riveda72** [reply]
I tried it but it doesn't work for me.
I've added this line into the config.txt file:
`"default_max_lora_number": 1...9,`
(yes, it's not the last line so there's the comma)

When I launch fooocus the config.txt is rejected and it loads only its default settings, giving me this error in CMD:
```
Failed to load config file "C:\Users\****\Fooocus\config.txt" . The reason is: Expecting ',' delimiter: line 55 column 30 (char 2659)
Please make sure that:
1. The file "C:\Users\****\Fooocus\config.txt" is a valid text file, and you have access to read it.
```
Col 30 in my txt file corresponds on the ":" right after "default_max_lora_number"
(I've masked my username with asterisks, of course there's my windows username there)

**Comment by Riveda72** [comment]
I tried it but it doesn't work for me.
I've added this line into the config.txt file:
`"default_max_lora_number": 1...9,`
(yes, it's not the last line so there's the comma)

When I launch fooocus the config.txt is rejected and it loads only its default settings, giving me this error in CMD:
```
Failed to load config file "C:\Users\****\Fooocus\config.txt" . The reason is: Expecting ',' delimiter: line 55 column 30 (char 2659)
Please make sure that:
1. The file "C:\Users\****\Fooocus\config.txt" is a valid text file, and you have access to read it.
```
Col 30 in my txt file corresponds on the ":" right after "default_max_lora_number"
(I've masked my username with asterisks, of course there's my windows username there)

**Reply by Rogala** [reply]
One number, 10, 100 or 1000

**Reply by Riveda72** [reply]
Do you mean a format like:
`"default_max_lora_number": 10,`
?

I got the same error

I also tried by adding more lora slots this way, the config.txt is correctly loaded but i still see only 5 slots in the web ui:
```
    "default_loras": [
        [
            "sd_xl_offset_example-lora_1.0.safetensors",
            0.1
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ],		
        [
            "None",
            1.0
        ]
    ],
```

**Reply by Rogala** [reply]
This line is enough
`"default_max_lora_number": 10,`
![image](https://github.com/user-attachments/assets/2d5080a5-04e2-44e1-811c-e9185ee30e03)
